runtime/doc/news.txt

```diff
@@ -316,6 +316,8 @@ The following new APIs and features were added.
 
 • Added |nvim_tabpage_set_win()| to set the current window of a tabpage.
 
+• Middle mouse button closes tabs.
```

Comment:
Middle mouse button clicking on what?

Reply:
`• Clicking on a tab with the middle mouse button closes it.`?

Reply:
```suggestion
• Clicking on a tabpage in the tabline with the middle mouse button closes it.
```